app/views/sa/expenses.scala.html

```diff
@@ -22,43 +22,78 @@
 
 @(appConfig: FrontendAppConfig)(serviceInfoContent: Html)(implicit request: Request[_], messages: Messages)
 
-@main_template(title = messages("sa.expenses.title"), appConfig = appConfig, bodyClasses = None, serviceInfoContent = serviceInfoContent) {
+@main_template(title = messages("expenses.title"), appConfig = appConfig, bodyClasses = None, serviceInfoContent = serviceInfoContent) {
 
-  <h1 id="expenses" class="heading-xlarge">@Messages("sa.expenses.heading")</h1>
-
-  <h2 class="margin-top-xlarge">@Messages("sa.expenses.what_expenses_can_i_include_in_my_sa_tax_return")</h2>
-  <p>@Messages("sa.expenses.you_put_all_of_your_allowable_expenses_")</p>
-  <p>@Messages("sa.expenses.you_can_choose_to_use_simplified_expenses_")</p>
-  <p>@Messages("sa.expenses.simplified_expenses_cannot_be_used_")</p>
-  <p>@Messages("sa.expenses.if_you_do_not_use_simplified_expenses_")</p>
-  <p>@Messages("sa.expenses.if_your_business_income_is_below_")</p>
+  <h1 id="expenses" class="heading-xlarge">@Messages("expenses.heading")</h1>
 
+<h2>@Messages("expenses.can_include")</h2>
+<p>@Messages("expenses.allowable_expenses")</p>
+<p>@Messages("expenses.send_proof")</p>
+<p>@Messages("expenses.keep_proof")</p>
+<p><a href="https://www.gov.uk/expenses-if-youre-self-employed" id="allowable-expenses" target="_blank"
+data-journey-click="link - click:Expenses : Find out more about allowable expenses if you’re self-employed">@Messages("expenses.find_out_more")</a></p>
 <div class="panel panel-border-wide">
- <p><a href ="https://youtu.be/FjXAH3tMg2I" id="what-expenses-can-i-include-in-my-sa-tax-return-video" target="_blank" data-journey-click="link - click:Expenses:Expenses if you are self employed - video link">@messages("sa.expenses.what_expenses_can_i_include")</a></p>
-  <p><a href="@controllers.routes.TranscriptController.onPageLoad("expenses-if-you-are-self-employed")" id="what-expenses-can-i-include-in-my-sa-tax-return-transcript" data-journey-click="link - click:Expenses:Expenses if you are self employed - video transcript">@Messages("sa.expenses.what_expenses_can_i_include_in_my_sa_tax_return") - @Messages("sa.expenses.video_transcript")</a></p>
+ <p><a href="https://youtu.be/FjXAH3tMg2I" id="what-expenses-video" target="_blank"
+ data-journey-click="link - click:Expenses : What expenses can I include in my Self Assessment tax return? - video">
+  @Messages("expenses.what_expenses_video")</a></p>
+ <p><a href="https://www.tax.service.gov.uk/business-account/help/transcript/expenses-if-you-are-self-employed" id="what-expenses-transcript"
```

Comment:
Hrefs for links pointing to pages within the service should use references to the controller methods, as they did before. Otherwise, these links will not work in the environments, any acceptance tests that check these links will fail.